### PR TITLE
chore(blog): add tauri-egui 0.1 announcement post

### PR DESCRIPTION
**Pull request by lucasfernog**

### CODE REVIEW

File: blog/2022-09-19-tauri-egui-0-1.mdx [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+tauri-egui = "0.1"
+```
+
+Now you need to enable the plugin:
+
+```rust
+fn main() {
+  tauri::Builder::default()
+    .setup(|app| {
+      app.wry_plugin(tauri_egui::EguiPluginBuilder::new(app.handle()));
```
Comment (amrbashir) [inline blog/2022-09-19-tauri-egui-0-1.mdx:30]: this is not relevant to the blog post but why is it named `wry_plugin` when it got nothing to do with `wry`?

**Review by amrbashir — APPROVED** [review]

File: blog/2022-09-19-tauri-egui-0-1.mdx (hunk 1 above)
Comment (lucasfernog) [reply blog/2022-09-19-tauri-egui-0-1.mdx:30]: it's a low level plugin (event loop) to the `tauri-runtime-wry` crate. `wry` exposes `tao`, so there's something to do with wry 😂 though i don't see any good name for this and too late to change it now.

Comment (amrbashir) [reply blog/2022-09-19-tauri-egui-0-1.mdx:30]: `event_loop_plugin` would be a good name but yeah too late now.

**Comment by FabianLars** [comment]
Do we want to update the banner at the top too?

**Comment by lucasfernog** [comment]
> Do we want to update the banner at the top too?

Too soon? Is there a good way to keep both announcements?

**Comment by FabianLars** [comment]
> Is there a good way to keep both announcements?

Probably not, [we're not even that much of a fan of a single one](https://github.com/tauri-apps/tauri-docs/pull/899#issuecomment-1248648317) 😅 

> Too soon?

Well, it would be ~3 days earlier than we had in mind, but at least the hero banner on the frontpage still shows 1.1.
idk, it's your call, we can change it later or even not at all 🤷

**Comment by FabianLars** [comment]
ah well, we could do something like "announcing the release of tauri 1.1 and tauri-egui 0.1"

**Comment by lucasfernog** [comment]
> ah well, we could do something like "announcing the release of tauri 1.1 and tauri-egui 0.1"

changed it. What do you think?

**Review by FabianLars — APPROVED** [review]

**Pull request merged**